src/components/DeployButton/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from "react";
+
+import { IDeployButton } from "../../interfaces/components/deployButton";
+import WalletService from "../../services/wallet";
+import { isHex } from "../../utils/isHex";
+
+function DeployButton({
+	chain,
+	isWalletConnected,
+	currentNetwork,
+	nativeBalance,
+	openModal,
+	bytecode,
+	homeErrorMessage,
+	setHomeErrorMessage,
+}: IDeployButton) {
+	const [valid, setValid] = useState<boolean>(false);
+
+	const handleDeployModal = async () => {
+		if (
+			isWalletConnected &&
+			currentNetwork.isSupported &&
+			currentNetwork.chainId === chain?.chainId
+		) {
+			openModal(chain);
+		} else if (
+			isWalletConnected &&
+			currentNetwork.isSupported &&
+			currentNetwork.chainId !== chain?.chainId
+		) {
+			WalletService.getInstance().switchNetwork({
+				chainId: chain?.chainId,
+				isSupported: true,
+			});
+		} else {
+			// eslint-disable-next-line no-console
+			console.log(chain.name);
```

Comment:
Should we remove this `console.log`?

Reply:
Removed the entire "else" statement, such that iff either of the "if" conditions are not met, the button does not do anything.